test/test-app/package.json

```diff
@@ -6,15 +6,18 @@
   "license": "MIT",
   "scripts": {
     "dev": "nodemon --ignore node_modules/ server.js",
-    "postinstall": "tsc greeting.ts",
-    "start": "echo 'starting node app' && node server.js"
+    "postinstall": "tsc greeting.ts && node-gyp -C addon configure build",
+    "start": "echo 'starting node app' && node server.js",
+    "compile": "node-gyp -C addon configure build"
```

Comment:
@danbev you've added a script target here, but it's not used. Did you mean for the `postinstall` to be `"tsc greeting.ts && npm run compile"`?

Reply:
Sorry, that was just a left over. I'll remove it.